src/command-bases/base.ts

```diff
@@ -32,7 +32,7 @@ export default abstract class CommandBase extends Command {
 
       return MuxCliConfigV1.check(configRaw);
     } catch (err) {
-      if (err.errno !== -2) {
```

Comment:
hmm, yeah, this is silly.

maybe we should just do `File.existsSync` instead of this kind of obscure thing?

Reply:
That would be even better I think; leave the abstraction to the Node.js layer

Reply:
Oh, and I'll be opting for the async version of that call ;-)  I don't want my hand smacked by the Node.js gods

Reply:
Alright, just made the update to use `fs.access()` and tested it as working! :D